frontend/amundsen_application/static/js/components/Badges/BadgeList/index.tsx

```diff
@@ -69,6 +69,11 @@ export default class BadgeList extends React.Component<BadgeListProps> {
       <span className="badge-list">
         {alphabetizedBadges.map((badge, index) => {
           let badgeConfig;
+          // search case
+          if (typeof badge === 'string' || badge instanceof String) {
+            const badgeName: string = badge.toString();
```

Comment:
Nit: I would remove this temporal variable and do `badgeConfig = getBadgeConfig(badge.toString());` directly